inheritance/index.js

```diff
@@ -0,0 +1,81 @@
+class Human {
+  constructor(options) {
+    this.name = options.name;
+    this.age = options.age;
+  }
+
+  sayHi() {
+    return 'Hello everybody!';
+  }
+}
+
+class Programmer extends Human {
+  constructor(options) {
```

Comment:
use destructuring and rest parameter in all similar cases as well as here

Reply:
I do not understand very well how to use Rest in this case